### PR TITLE
Add more information to the knowledge graph docs

### PR DESCRIPTION
**Pull request by jonathanglassman**
- Add information on how the knowledge graph is built
- Add information on the nodes and relationships
- Add extra intermediate data file on external links

### CODE REVIEW

File: source/analysis/govgraph/nodes-relationships/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| ContentID (Cid) | ContentID (Cid) | HYPERLINKS _TO | States if there is a hyperlink from the source node to the target node. |
+| ContentID (Cid) | ContentID (Cid) | USER _MOVEMENT | Using (weighted) functional edges, captures the number of user movements from the source node to the target node (content items) over the previous 3 weeks (ending 2 days prior to the current date). |
+| ContentID (Cid) | ContentID (Cid) | HAS _SUGGESTED _ORDERED _RELATED _ITEMS | The target node is an algorithmically-generated related link on the source node. |
+| ContentID (Cid) | ContentID (Cid) | HAS _SIMILAR _CONTENT | Based on old data.<br>Highlight that source and target content ID have similar content. |
+| Cid | Chapter | HAS _CHAPTER | Some pages of type “guide” are split into chapters.<br>This indicates that a given content ID for the source node contains the target content item as one of its chapters. |
+| Cid | ExternalPage | HYPERLINKS _TO | States that a page on `www.gov.uk` links to an external page (for example, `assets.digital.cabinet-office.gov.uk`). |
+| Cid | Taxon | IS _TAGGED _TO | Some content IDs are linked to taxons, for example, “Benefits entitlement” or “Welfare”. |
```
Comment (exfalsoquodlibet) [inline source/analysis/govgraph/nodes-relationships/index.html.md.erb:22]:
```suggestion
| Cid | Taxon | IS _TAGGED _TO | Content items that contain a link to a given Taxon page. For example, the page `www.gov.uk/child-tax-credit` contains a link to the Taxon page "Tax  Credits" (`www.gov.uk/topic/benefits-credits/tax-credits`). |
```

I think this makes it clear what exactly is linked to what.

File: source/analysis/govgraph/nodes-relationships/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+| Cid | Step | HAS _STEP | Step-by-step pages on GOV.UK are divided in steps. |
```
Comment (exfalsoquodlibet) [inline source/analysis/govgraph/nodes-relationships/index.html.md.erb:23]: Could we add a reference to this https://data-community-tech-docs.london.cloudapps.digital/analysis/govgraph/int-data-files/#step-nodelist-cid-to-step-edgelist-csv in the description?
It'll help clarify what the relationship actually captures.

File: source/analysis/govgraph/nodes-relationships/index.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| Step | Cid | HAS _TASK | Each step in a step-by-step page includes a link to another page, also known as a task. |
```
Comment (exfalsoquodlibet) [inline source/analysis/govgraph/nodes-relationships/index.html.md.erb:24]: Similarly here, if we could reference https://data-community-tech-docs.london.cloudapps.digital/analysis/govgraph/int-data-files/#task-cid-is-part-of-step-edgelist-csv

**Review by exfalsoquodlibet — CHANGES_REQUESTED** [review]
Thanks @jonathanglassman. A couple of (minor) points from me to consider.

File: source/analysis/govgraph/index.html.md.erb [hunk 4]
```diff
@@ -56,411 +63,26 @@ This network visualises user movement between specified content items (pages), w
...
-| role_content_id | Content ID of the role (does not map to a page) | 5f35e375-7362-46a9-bb7a-abdfc64bb839 |
-| role_title | Name of the role| Pubs Code Adjudicator|
-
-### roles_to_organisations_edgelist.csv
-
-A list of all the role-organisation relationships.
+There is one instance of the Knowledge Graph, the [user-facing Knowledge Graph](https://knowledge-graph.integration.govuk.digital:7473/browser/) that content designers use to answer common content questions.
 
-This file is created by the [`extract_publishing_api.py` Python script](https://github.com/alphagov/govuk-knowledge-graph/blob/master/src/data/extract_publishing_api.py) and has the following data fields.
+1. To create the Knowledge Graph instance, every day at 5am the following applications are automatically run in sequence:
```
[the remark this answers is not in the document]
Comment (jonathanglassman) [reply source/analysis/govgraph/index.html.md.erb:68]: whoops, good spot!

File: source/analysis/govgraph/int-data-files/index.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,401 @@
...
+
+# GOV.UK Knowledge Graph intermediate data files
+
+The following intermediate data files are used to build the Knowledge Graph instance.
+
+## preprocessed_content_store_<DDMMYY>.csv.gz
+
+A data set of most GOV.UK content pages with multiple attributes for each page.
+
+This data set is made of Content Store data, but excludes:
```
[the remark this answers is not in the document]
Comment (jonathanglassman) [reply source/analysis/govgraph/int-data-files/index.html.md.erb:16]: whoops, good spot

**Review by exfalsoquodlibet — APPROVED** [review]
Thanks for your changes @jonathanglassman. Happy with it.

**Pull request merged**